test/fixtures/short_game_teams.csv

```diff
@@ -0,0 +1,4 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```

Comment:
fixtures files for testing

Reply:
I did not create a fixture file for testing game class, but called full CSV. It takes a long time and I will do this now!